spec/features/address_bar_spec.rb

```diff
@@ -132,6 +132,48 @@ def query_string
         expect(page.current_url).not_to have_content("sb=0%2C0%2C10%2C10")
       end
```

Comment:
we should add an ```after :all do``` block to re-set the bounding_box for use in your new test. If the clearing filter test (line 125) is executed before the new test, the new one will not have a collection-level filter bounding box to begin with.

```ruby
after :all do
  visit '/search'
  wait_for_xhr
  create_bounding_box(0, 0, 10, 10)
  wait_for_xhr
end
```